app/components/common/eventCard/index.tsx

```diff
@@ -25,8 +23,7 @@ export default function EventCard({
 
         <div className="ml-6 bottom-4 gap-2 absolute flex flex-col">
           <div className="text-xs text-gray-700 font-medium">
-            {' '}
-            {fromTime} to {toTime}{' '}
+            {fromTime} to {toTime}
```

Comment:
Are we sure we will get the exact required format? We don't require any formating of time?